### PR TITLE
Fix: ensure media labels are loaded when opening a draft in offline mode

### PR DESCRIPTION
**Pull request by eyelidlessness**
Manually verified, there are no tests of controller-webform.js and it has the same testing challenges as enketo-webform\*

### CODE REVIEW

**Comment by yanokwa** [comment]
@eyelidlessness Is this a regression? If so, when did the issue start?

**Comment by eyelidlessness** [comment]
Yes, this was a regression from the original last-saved work. Listening for `FormReset` was removed in [this commit](https://github.com/eyelidlessness/enketo-express/commit/2cefa10735b63fadf732e23ef541f8502956b15e#diff-c2d218e1091faeffd002d2a2e70688a3146214251393d5955ea5d13863ed44a5L44) and merged in #269. During review, we discussed the fact that the indirection of using events was confusing and hard to follow, and decided to:

1. Make the `formCache.updateMedia` call directly rather than depend on events
2. Preserve the event dispatch in case forks may depend on it for some other reason

I was surprised not to find this discussion in #269, but found discussion of it in Slack history. That change also produced [this comment](https://github.com/enketo/enketo-express/pull/269/files#diff-c00aae18f8d960daf04e51f849c2315a43fa453891b7b40e91cf12dd84a1e26bR180-R193) for posterity (which has since been updated/corrected). But somehow I just... missed the fact that the same event was being dispatched in `_loadRecord`.

It's likely this regression has been there since #269, it's surprising that we're only now discovering it but I suspect that's because so many other things around media URLs have been broken in that time, it may have just obscured this additional issue until the various other fixes were in.
